src/registrar/config/urls.py

```diff
@@ -58,6 +59,16 @@
 
 urlpatterns = [
     path("", views.index, name="home"),
+    path(
+        "organization/<int:portfolio_id>/domains/",
```

Comment:
Do we want to call it organization or portfolio? We do have "organization" mentioned in a lot of places (domain requests & info) and everyone really is part of an organization. Should we instead call it portfolio, since that is the name we were trying to go with to avoid confusion?